utils.py

```diff
@@ -1,10 +1,10 @@
+# import necessary libraries
 import pandas as pd
 from typing import List, Tuple, Union
 import mysql.connector as mysql
 
-#function that connects to MySQL and creates a cursor object
-
-def connect_to_mysql(host, user, password):
+# Define a function that connects to a MySQL server and creates a cursor object.
+def connect_to_mysql(host: str, user: str, password: str) -> Tuple[mysql.connection.MySQLConnection, mysql.cursor.MySQLCursor]:
```

Comment:
`mysql.connection.MySQLConnection, mysql.cursor.MySQLCursor` these are not datatypes. These are Class

Reply:
since cursor was defined inside a function, I couldn't take that here in the create_database function, how to resolve it?

Reply:
actually it should be what you had earlier, I asked chatGPT & on their documentation too what it should be, looks like its the same